src/editors/starrating.js

```diff
@@ -30,7 +30,7 @@ JSONEditor.defaults.editors.starrating = JSONEditor.defaults.editors.string.exte
 
     for(var i = this.enum_values.length-1; i>-1; i--) {
```

Comment:
Just asking ...

```
for(var i = this.enum_values.length; i>0; i--) {

  var id = this.formname + i;
```

Wouldn't that be more understandable?

Reply:
True. I have changed it to your suggestion.